CanonicalLtd/maas-docs
# Restructure metadata files for v1.0.0

Version 1.0.0 of documentation-builder is almost here.

This restructures and metadata slightly to fit in with its expected format.

QA
--

Install `1.0.0-rc2`:

``` bash
snap remove documentation-builder  # If it's installed
snap install --channel candidate documentation-builder
```

Now run `documentation-builder` in the root directory. It should still build beautifully.
